torch/_inductor/codecache.py

```diff
@@ -861,6 +857,8 @@ def get_include_and_linking_paths(
         # For those cases, include the lpath and libs command as we do for pytorch above.
         # This approach allows us to only pay for what we use.
         ipaths = cpp_extension.include_paths(cuda) + [sysconfig.get_path("include")]
+        if aot_mode:
```

Comment:
I don't think we will get to `else` at line 854 for our use case?

Reply:
We will get to the ```else``` branch. I hit this with the internal e2e test, where none of clauses from ```include_pytorch```, ```vec_isa != invalid_vec_isa```, ```cuda``` and ```config.cpp.enable_kernel_profile``` is True.

Reply:
`vec_isa == invalid_vec_isa` sounds problematic if we are talking about optimizing cpu models?